### PR TITLE
turn-by-turn navigation with Google Maps

### PR DESCRIPTION
**Pull request by calvinschmidt**
I now integrated a basic turn-by-turn navigation, using Google Maps. I adjusted the mapStyle as well to fit our app design. For a testing route, the following coordinate is a good start point:

latitude: 52.613632,
longitude: 13.47463

Thanks for reviewing!

### CODE REVIEW

File: components/MapRoute.js [hunk 1]
```diff
@@ -235,41 +251,89 @@ class MapRoute extends React.Component {
...
-    const { coordinates, focusedLocation, isMapReady } = this.state;
-
     return (
       <View style={styles.container}>
         <MapView
+          provider="google"
           style={styles.map}
-          initialRegion={focusedLocation}
           showsUserLocation
           followsUserLocation={Platform.OS === 'ios'}
```
Comment (nschoen) [inline components/MapRoute.js]: `followsUserLocation={Platform.OS === 'ios'}` can be removed, as now `checkUserLocation` navigates to the user position for iOS as well.

**Review by nschoen — APPROVED** [review]
Looks very good, thanks. Just two things:

`followsUserLocation={Platform.OS === 'ios'}` can be removed from the MapView, as `checkUserLocation` already keeps track of the GPS position.

And in the iOS simulator, the GPS position and the Interchange position may not fit in the initial view:

File: components/MapRoute.js (hunk 1 above)
Comment (calvinschmidt) [reply components/MapRoute.js]: Done.

**Comment by calvinschmidt** [comment]
I adjusted the map, so the GPS position and the Interchange position now fit in the initial view.

**Pull request merged**